cirq/aqt/aqt_simulator.py

```diff
@@ -0,0 +1,89 @@
+import json
+from typing import Union
+from cirq import Circuit, Simulator, LineQubit, study
+from cirq import measure, X, Y, XX
+from cirq.aqt.aqt_device import default_noise_dict
+"""Simulator for the AQT ion trap device"""
+
+gate_dict = {'X': X, 'Y': Y, 'MS': XX}
+
+
+class AQTSimulator:
```

Comment:
I think this should be `AQTNoiseModel` and/or `AQTDevice` instead of `AQTSimulator`